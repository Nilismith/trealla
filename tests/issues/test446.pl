:- initialization(main).

main :-
	([c|Y],b,a) = (X,Y), write({X,Y}),nl.


