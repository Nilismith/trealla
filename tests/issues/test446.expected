{[c|(b,a)],b,a}
